README.md

```diff
@@ -10,14 +10,100 @@ and ease of use. Try our online demos:
 [LLaMA2](https://huggingface.co/spaces/lmz/candle-llama2),
 [yolo](https://huggingface.co/spaces/lmz/candle-yolo).
 
+## Installation
```

Comment:
I feel most of this belongs to the book which would test the examples actually.

Also we should link the book from the README.

Reply:
Sure happy to move the installation section to the book if you prefer and instead write something like: 

"Make sure to install `cargo` as described in the book"

For PyPI packages pretty much every GitHub README has an installation section which I like a lot because the reader doesn't have to click through too much to get started. Is this different for the Rust community?

Reply:
Well, it's always `cargo add MYPACKAGE` or `cargo run` from within the package.
It's standard enough there's nothing really. Unless you're building a binary (which is not the case here).

https://github.com/tokio-rs/tokio
https://github.com/rust-lang/regex
for instance.

Usually the getting started is more the snippet of code of usage. And anything which is not obvious, classical pitfalls.

The docs are almost always in docs.rs https://docs.rs/candle-core/0.1.2/candle_core/

Then for more complex doc, there's no 1 stop shop solution, but I find the `mdbook` relatively broadly used and a pleasure to work with (good design, automatically tested snippets, search integrated). The most widely used ressource for *learning* rust itself is written in this : https://doc.rust-lang.org/book/ :)